### PR TITLE
[Monitor Vlan] Fix a typo in hostif

### PR DESCRIPTION
**Pull request by prsunny**
**What I did**
hostif_name -> host_ifname as per the schema definition

**Why I did it**
Fix typo

**How I verified it**

**Details if related**

### CODE REVIEW

File: tests/dvslib/dvs_vlan.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def create_vlan(self, vlan):
 
     def create_vlan_hostif(self, vlan, hostif_name):
```
Comment (sumukhatv) [inline tests/dvslib/dvs_vlan.py:16]: do we want to change this hostif_name too?

Comment (prsunny) [reply tests/dvslib/dvs_vlan.py:16]: Not modifying the param names but just the field name

**Review by sumukhatv — APPROVED** [review]

**Comment by lguohan** [comment]
@prsunny , please submit the schema change to yang model as well.

**Comment by abdosi** [comment]
@prsunny please create pr for 201911. Cherry-pick has conflict.

**Comment by abdosi** [comment]
@prsunny have resolve the conflict manually. DVS test is not there in 201911 for this feature as previous PR we skipped dvs test as part of conflict

**Pull request merged**